Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Post Item/Post Types/Headline Post.swift

```diff
@@ -52,7 +52,7 @@ struct HeadlinePost: View {
                             StickiedTag(tagType: .community)
                         }
                         
-                        Text(postView.post.name)
+                        Text(verbatim: postView.post.name)
```

Comment:
That's not the whole story, since iOS 15 Text(_: String) uses AttributedString to display basic Markdown! but the profiler told me that someone was using/parsing a little too many AttributedStrings so the Text(verbatim: String) initialiser tells Text to not to _any_ text processing